DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1070 @@ Live discussion in #document-authoring on Slack.
+
+: `reference_count`
+:: Total number of [=media segments=] referenced by the index segment.
+
+: `reference_type`
+:: `0`
+
+: `referenced_size`
+:: Size of the [=media segment=] in bytes. [=Media segments=] are assumed to be consecutive, so this is also the distance to the start of the next [=media segment=].
+
+: `subsegment_duration`
+:: Duration of the [=media segment=] in [=timescale units=].
+
+: `starts_with_SAP`
+:: `1`
+
+: `SAP_type`
+:: Either `1` or `2`, depending on the sample structure in the [=media segment=].
+
+: `SAP_delta_time`
+:: `0`
+
+#### Moving the period start point (indexed addressing) #### {#timing-addressing-indexed-startpoint}
+
+When splitting periods in two or performing other types of editorial timing adjustments, a service might want to start a period at a point after the "natural" start point of the [=representations=] within.
+
+For [=representations=] that use [=indexed addressing=], perform the following adjustments to set a new [=period=] start point:
+
+1. Update `SegmentBase@presentationTimeOffset` to indicate the desired start point on the [=sample timeline=].
+1. Update `Period@duration` to match the new duration.
+
+### Explicit addressing ### {#timing-addressing-explicit}
+
+A representation that uses <dfn>explicit addressing</dfn> consists of a set of [=media segments=] accessed via URLs constructed using a template defined in the MPD, with the exact time span covered by each [=media segment=] described in the MPD.
+
+Clauses in section only apply to [=representations=] that use explicit addressing.
+
+<figure>
+	<img src="Images/Timing/ExplicitAddressing.png" />
+	<figcaption>Explicit addressing uses a segment template that is combined with explicitly defined time spans for each [=media segment=] in order to reference [=media segments=], either by start time or by sequence number.</figcaption>
+</figure>
+
+The MPD SHALL contain a `SegmentTemplate/SegmentTimeline` element that contains a set of [=segment references=] which satisfies the requirements defined in this document. The references exist as a sequence of `S` elements, each of which references one or more [=media segments=] with start time `S@t` and duration `S@d` [=timescale units=] on the [=sample timeline=]. The `SegmentTemplate@duration` attribute SHALL NOT be present.
+
+To enable concise reference definitions, an `S` element may be a repeating [=segment reference=] that indicates a number of repeated consecutive [=media segments=] with the same duration. The value of `S@r` SHALL indicate the number of additional consecutive [=media segments=] that exist.
+
+Note: Only additional references are counted, so `S@r=5` indicates a total of 6 consecutive [=media segments=] with the same duration.
+
+The start time of a [=media segment=] SHALL be calculated from the start time and duration of the previous [=media segment=] if not specified by `S@t`. There SHALL NOT be any gaps or overlap between [=media segments=].
```

Comment:
From @haudiobe: What does it mean: "The start time of a media segment SHALL". It is a definition, not a requirement

Reply:
What is a definition and what is a requirement is not a big difference in my mind. Different readers have different viewpoints. Clearly stating things we expect to be constants in the universe using SHALL statements makes sense to me and avoids alternative interpretations or doubt from easily arising.